src/api/clearlyDefined.js

```diff
@@ -60,10 +66,10 @@ export function getCuration(token, entity, params = {}) {
 }
 
 /**
- * List all of the curations (if any) using the given coordinates as a pattern to match
+ * List all of the curations (if any) using the given coordinates as a pattern to match, despite the revision
  * @param  {} token
  * @param  {} entity
- * @param  {} params
+ * @param  {object} params can contain property: pendingPrs, if true returns also curations not already merged
```

Comment:
Not a fan of boolean arguments. better would be an "expand" enum that indicated what all to include. that is the way that the REST api would be written.

Reply:
Yes, that is a correct interpretation for the REST api. The `getCuration` function should be similar. That is, in the future the REST api will allow `expand=prs,foos,bars`. At that time, what will `getCuration` look like? will it have prs, foos and bars as `params` and then a whole mess of tests to build up the REST query param list?

Instead, I suggest that `getCuration` take `params` as  { expand: ['prs'], state: 'pending' }. That elminates the plague of boolean params and makes the code API more similar to the REST API (and thus easier to doc and understand).

Reply:
Ok, i see. Having the big picture of the future of the REST API gives to the implementation a different approach.
I will change this following your suggestions.